### PR TITLE
Add Eli to README

### PR DESCRIPTION
**Pull request by easachs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ I am adding text
 #notes:
 
 Might need to redo Turing folder? (git init at the root of you r project not the root of the entire folder)
+
```
Comment (SandyyMarie) [inline README.md:7]: Ooo this leaves a comment!

Comment (SandyyMarie) [reply README.md:7]: Oops this is a review

**Review by SandyyMarie — COMMENTED** [review]
Done reviewing the empty line!

File: README.md (hunk 1 above)
Comment (easachs) [reply README.md:7]: Leaving a note here...

File: README.md [hunk 2]
```diff
@@ -4,3 +4,5 @@ I am adding text
 #notes:
 
 Might need to redo Turing folder? (git init at the root of you r project not the root of the entire folder)
```
Comment (SandyyMarie) [inline README.md:6]: I deleted this in future branches

**Review by SandyyMarie — COMMENTED** [review]
Done playing with reviews ty

**Pull request merged**